sdk/src/main/java/io/opentelemetry/sdk/trace/export/ExporterConstants.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace.export;
+
+import java.util.concurrent.TimeUnit;
+
+/** Provides constants for all exporter default configuration values. */
+public class ExporterConstants {
+
+  private ExporterConstants() {}
+
+  /** common unknown constant. */
+  public static final String UNKNOWN = "unknown";
```

Comment:
This seems like a too generic constant, we'd generally expect the default configuration value to be empty. Translating that to unknown may be an exporter-specific implementation detail.

Reply:
thx. fixed at lastest commit.